src/darwin/Framework/CHIP/MTRCommissioningParameters.h

```diff
@@ -113,6 +114,13 @@ MTR_AVAILABLE(ios(16.1), macos(13.0), watchos(9.1), tvos(16.1))
  */
 @property (nonatomic, copy, nullable) NSNumber * acceptedTermsAndConditionsVersion MTR_PROVISIONALLY_AVAILABLE;
 
+/**
+ * List of attribute paths to read from the commissionee (in addition to
+ * whatever attributes are already read to handle readEndpointInformation being
+ * YES, or to handle other commissioning tasks).
+ */
+@property (nonatomic, copy, nullable) NSArray<MTRAttributeRequestPath *> * extraAttributesToRead MTR_PROVISIONALLY_AVAILABLE;
```

Comment:
Should this be `MTR_UNSTABLE_API` ? I thought we used `MTR_PROVISIONALLY_AVAILABLE` for stuff that's provisional in the spec. At least that was the original intent, but I don't know if we've actually followed that distinction.

Reply:
I just forgot about MTR_UNSTABLE_API.  This should absolutely be MTR_UNSTABLE_API.  Done.